e2e_test/source/cdc_paid/cdc.load.slt

```diff
@@ -0,0 +1,74 @@
+# CDC source basic test
+control substitution on
+
+# TODO(kexiang): add more tests here
+# Some columns missing and reordered (sql-server-cdc)
+
+statement ok
+CREATE TABLE orders (
```

Comment:
We can leave all the test refactoring to the next PR.

Reply:
Why next PR? I'm very reluctant to have improper tests committed into the repo (more reluctant than leaving a refactoring to the next PR). It would be very harmful if not cleaned in time. Simply deleting the tests might even be better than leaving them as-is. 😄

Reply:
I hope the PR can be included in the next release, as we are already close to the code freeze date. The delay caused by the refactoring, as well as the asynchronous code review and revision process, means that the refactorization will take some time to complete. This may prevent the PR from being merged before the code freeze date.

Reply:
> since you already implemented the shared source in the PR, what about only provide the shared source syntax?

I'll remove the table with connector part tomorrow.

Reply:
> I hope the PR can be included in the next release, as we are already close to the code freeze date.

It makes sense, but I still wouldn't agree to commit the tests for the reasons I said above.

I can come up with a workaround: Move the current tests to PR2 on top of this PR, then we don't need to merge the tests immediately, but can still ensure PR1 passed the tests there.

Reply:
Good idea. Will conduct the movement tmr.

Reply:
To clarify, I would agree to delay the code refactoring (e.g., move sql-server specific code out of `common`) in later PRs, but I really don't want to increase the complexity of CI & test scripts, even temporarily.

Reply:
Updated. PTAL